consensus_message

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 REMME
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message ConsensusMessagePayload {
+  // period of consensus loop
+  uint64 period = 1;
+  
+  // header_signature of block that was proposal
+  string proposal_id = 2;
+  
+  // address of node that propose this header_signature
+  string voter_id = 3;
+}
+
+message ConsensusMessage {
+// All kind of consensus message
+  enum Type {
+      VALUE_PROPOSAL = 0;
+      SOFT_VOTES = 1;
+      CERT_VOTES = 2;
+      NEXT_VOTES = 3;
+  }
+  
+  // type of consensus message
+  Type type = 1;
+  
+  // payload of message
+  MessagePayload payload = 0;
```

Comment:
Please write field ids in the same order you introduce them in this file. Also 0 is not a valid field id.